ml-agents/mlagents/trainers/ppo/models.py

```diff
@@ -1,12 +1,115 @@
 import logging
 import numpy as np
+import sys
 
 import tensorflow as tf
 from mlagents.trainers.models import LearningModel
 
+# Variable scope in which created variables will be placed under
+TOWER_SCOPE_NAME = "tower"
+
 logger = logging.getLogger("mlagents.trainers")
 
 
+class PPOMultiGPUModel(object):
+    def __init__(
+        self,
+        brain,
```

Comment:
Seems like you should be able to use `*args` and `**kwargs` and pass these through to prevent duplication, right?  The only argument specific to this model is `devices`.  So then you can generically pass in the args so we don't have to change both places when we change PPO.